cmd/build.go

```diff
@@ -31,10 +31,9 @@ func cmdBuild(cmd *cli.Cmd) {
 	defaultOpts(&opts, cmd)
 	cmd.BoolOptPtr(&opts.dryRun, "dry-run", false, "Simulate what would happen without actually doing anything dangerous.")
 	cmd.BoolOptPtr(&opts.forceRebuild, "force-rebuild", false, "Forces rebuilding the entire image graph, without regarding if the target version already exists.") //nolint:lll
-	cmd.BoolOptPtr(&opts.generateGraph, "g graph", false, "Instruct dib to generate graphviz during the build process.")
-	cmd.BoolOptPtr(&opts.runTests, "t test", false, "Instruct dib to run goss tests during the build process.")
-	cmd.BoolOptPtr(&opts.retagLatest, "retag-latest", false,
-		"Should images be retagued with the 'latest' tag for this build")
+	cmd.BoolOptPtr(&opts.disableGenerateGraph, "disable-graph", false, "Instruct dib to disable generation of graph during the build process.")                     //nolint:lll
+	cmd.BoolOptPtr(&opts.disableRunTests, "disable-test", false, "Instruct dib to disable execution of Goss tests during the build process.")                       //nolint:lll
```

Comment:
What do you think of even shorter names for the flags ?

```suggestion
	cmd.BoolOptPtr(&opts.disableGenerateGraph, "no-graph", false, "Disable generation of graph during the build process.")                     //nolint:lll
	cmd.BoolOptPtr(&opts.disableRunTests, "no-tests", false, "Disable execution of tests during the build process.")                       //nolint:lll
```